.gitattributes

```diff
@@ -1,6 +1,7 @@
 *.csv eol=lf
 *.js eol=lf
 *.jsx eol=lf
+*.fix eol=lf
```

Comment:
@IllusionMH in case this is relevant to your rules... this took me a bit of frustration to get to.